index.html

```diff
@@ -135,28 +171,45 @@ <h4>Don't Forget These!</h4>
           <li class="list-group-item">Vestibulum at eros</li>
         </ul>
       </div>
+      </div>
     </div>
     <div class="row">
-      <div class="col-sm-12">
-        <form>
+      <div class="col-8 col-sm-6 col-md-6 col-lg-6">
+
+
+      <form>
+        <div class=form-group>
```

Comment:
Here we want to wrap each individual label/input pair in a div with `form-group` as the class, then wrap all of the radio buttons in another one. The wording in the instructions is a little bit ambiguous!